source/docs/solar/nsrdb/nsrdb-GOES-aggregated-v4-0-0-download.html.md.erb

```diff
@@ -0,0 +1,312 @@
+---
+title: "Nsrdb Goes Aggregated V4.0.0"
+summary: "Collect and download, as CSV, a configurable set of solar and meteorological data fields from The NSRDB. The National Solar Radiation Database [(NSRDB)](https://nsrdb.nrel.gov/) is a serially complete collection of satellite-derived measurements of solar radiation—global horizontal, direct normal, and diffuse horizontal irradiance—and meteorological data. These data have been collected at a sufficient number of locations and temporal and spatial scales to accurately represent regional solar radiation climates. The data are publicly available at no cost to the user. These API provide access to downloading the data. Other options are detailed [here](https://nsrdb.nrel.gov/data-sets/how-to-access-data). Read more about the datasets at [https://nsrdb.nrel.gov](https://nsrdb.nrel.gov)
+detail: This API is capable of creating very large downloadable archives. Unlike a typical API, the response to this provides a request acknowledgement. The real work of creating the requested downloadable archive will continue to run on the server side. When complete, and email will be sent to the email address provided in the initial request with a link to a file to download. In addition to this two step workflow, there is also an option for users who wish to download a single CSV file in direct response to an API request. The <em>.csv</em> format may be used to download a CSV directly. This feature is restricted to use with only a single POINT, for a single YEAR at a time."
```

Comment:
You need to close your quotes here, and open them up again after `detail:` to get the formatting correct. When you run it locally, check that it matches the formatting of [this one](http://localhost:4480/docs/solar/nsrdb/psm3-2-2-download/) where the detail is a distinct separate paragraph. Also on the previous page you can see how to detail doesn't show up in the summary blurb:
![Screenshot 2024-10-31 at 12 58 36 PM](https://github.com/user-attachments/assets/2aac5583-3dc3-461b-a50c-b9e83fcf8a8e)

Whereas on yours, not only does the detail show up, but the word "detail:" also shows up:

![Screenshot 2024-10-31 at 1 00 10 PM](https://github.com/user-attachments/assets/1091c46f-f25f-4cb6-9fe7-19c67b45e13f)

Reply:
Thanks for catching this! [Resolved](https://github.com/NREL/developer.nrel.gov/pull/365/commits/bd86d3bb1e13c5b47bc6e1fc10298ef4e9194651)